src/main/java/io/swagger/api/watchlist/WatchlistApiController.java

```diff
@@ -82,7 +88,18 @@ public ResponseEntity<Watchlist> watchlistWatchlistIdVisibilityPut(@Parameter(in
             return ResponseEntity.notFound().build();
         }
 
+        AddHateoasUrls(watchlist);
         return ResponseEntity.ok(watchlist);
     }
 
+    private void AddHateoasUrls(Watchlist watchlist)
```

Comment:
Luckily, all of the Watchlist APIs just return the new state of the entire watchlist. So they all call this function.